modules/ocf_hpc/templates/slurmdbd.conf.erb

```diff
@@ -0,0 +1,24 @@
+ArchiveEvents=yes
+ArchiveJobs=yes
+ArchiveResvs=yes
+ArchiveSteps=no
+ArchiveSuspend=no
+#ArchiveScript=/usr/sbin/slurm.dbd.archive
+#AuthInfo=/var/run/munge/munge.socket.2
+AuthType=auth/munge
+DbdHost=<%= @hostname %>
+#DbdBackupHost=
+DebugLevel=4
+PurgeEventAfter=1month
+PurgeJobAfter=12month
+PurgeResvAfter=1month
+PurgeStepAfter=1month
+PurgeSuspendAfter=1month
+LogFile=/var/log/slurm-llnl/slurmdbd.log
+#PidFile=/var/tmp/jette/slurmdbd.pid
+SlurmUser=slurm
+StorageType=accounting_storage/mysql
+StorageUser=ocfhpc
+StoragePass=<%= @slurmdbd_mysql_password %>
+StorageHost=mysql.ocf.berkeley.edu
```

Comment:
Can probably just do `mysql` here instead of putting the whole FQDN if you'd like to be a little less verbose, but both will work.